src/requirements/data/index.ts

```diff
@@ -357,6 +356,11 @@ const json: RequirementsJson = {
       schools: ['HE'],
       requirements: hdMinorRequirements,
     },
+    INEQ: {
```

Comment:
Similarly - could you please revert the changes in this file? Let me know if you need any help with how to do so.